addons/website_sale/static/src/js/website_sale.js

```diff
@@ -684,6 +684,34 @@ publicWidget.registry.WebsiteSaleLayout = publicWidget.Widget.extend({
     },
 });
 
+publicWidget.registry.WebsiteSaleAccordionProduct = publicWidget.Widget.extend({
+    selector: "#product_accordion",
+
+    /**
+     * @override
+     */
+    async start() {
+        await this._super(...arguments);
+        this._updateAccordionActiveItem();
+    },
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
```

Comment:
Just sort methods by call order; it's totally useless to separate public and private methods.
```suggestion
```